.changeset/nervous-pets-sleep.md

```diff
@@ -0,0 +1,116 @@
+---
+'@primer/react': major
+---
+
+![image](https://user-images.githubusercontent.com/2313998/155632630-60822d7b-6053-480d-925d-cea4d3b04919.png)
+
+The `CheckboxGroup` and `RadioGroup` components will be used to deprecate the `ChoiceFieldset` component. The deprecation of `ChoiceFieldset` also allows us to deprecate `ChoiceInputField`.
+
+`CheckboxGroup` and `RadioGroup` have the ability to render contextual content with your fieldset: labels, validation statuses, captions. It also handles the ARIA attributes that make the form controls accessible to assistive technology.
+
+<table>
+<tr>
+<th> Before </th> <th> After </th>
+</tr>
+<tr>
+<td valign="top">
+
+```jsx
+import {ChoiceFieldset} from "@primer/react";
+
+// Multi-select
+<ChoiceFieldset>
+  <ChoiceFieldset.Legend>Prefered Primer component interface</ChoiceFieldset.Legend>
```

Comment:
The before / after is overflowing quite a lot.. is it possible to add more line breaks to the `before` snippet so it brings after more into view?

```suggestion
  <ChoiceFieldset.Legend>
    Prefered Primer component interface
  </ChoiceFieldset.Legend>
```